examples/77_blackwell_fmha/collective/fmha_fusion.hpp

```diff
@@ -204,14 +204,10 @@ struct CausalMask : NoMask {
     // See note below on different ways to think about causal attention
     // Again, we'd add the offset_q into the max_blocks_q calculation
     int max_blocks_k = Base::get_trip_count(blk_coord, tile_shape, problem_size);
-    if constexpr (IsQBegin) {
-      int max_blocks_q = ceil_div((get<0>(blk_coord) + 1) * get<0>(tile_shape), get<1>(tile_shape));
-      return std::min(max_blocks_k, max_blocks_q);
-    } else {
-      const int offset_q = get<1>(problem_size) - get<0>(problem_size);
-      int max_blocks_q = ceil_div((get<0>(blk_coord) + 1) * get<0>(tile_shape) + offset_q, get<1>(tile_shape));
-      return std::min(max_blocks_k, max_blocks_q);
-    }
+    int offset_q = IsQBegin ? int(get<1>(problem_size)) - int(get<0>(problem_size)) : 0;
```

Comment:
this appears to be doing the opposite of what's on main branch? Should this be
```suggestion
    int offset_q = IsQBegin ? 0 : int(get<1>(problem_size)) - int(get<0>(problem_size)) ;
```
and so on for the other lines